### DISCUSSION TITLE
增加对于NeoForge的支持

### DISCUSSION THREAD

**Discussion by NBWSLeee** [Meloong-Git/PCL#2365] [提问] [answered]
一个新出的模组加载器，从Forge分支出来的。Forge几乎所有的开发者都去了NeoForged团队，所以是很有前途的。

**Comment by WForst-Breeze** [comment, marked as answer]
> 猫曰：先多等等看看情况.jpg

#2244

**Comment by iPodtouch6557** [comment]
> Forge几乎所有的开发者都去了NeoForged团队

那么原来的forge咋办（doge）是不是意味着开发速度会慢

**Reply by yaxuan97** [reply]
原来的Forge已经停留在了1.20.1 - 47.1.43版本没有更新了
